.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - "3.5"
+before_install:
+  - export DISPLAY=:99.0
```

Comment:
why do we need as display? For matplotlib for creating the QLA plots? In this case, let's rather use another backend and make sure shifthelper works without a display. People running this shifthelper somewhere do not expect it do need a screen, so we should not need one. 
I can open an issue for this, no need to do it right away.